generator/src/main/java/com/algorand/sdkutils/listeners/GoGenerator.java

```diff
@@ -326,6 +328,7 @@ private StringBuffer spaces (int c) {
         }
```

Comment:
Either return a string, or use `StringBuilder`, no need for this helper to decide whether or not the caller needs thread safety

Reply:
fixed.